pkg/client/client_test.go

```diff
@@ -30,11 +30,11 @@ import (
 )
 
 const (
-	testVaultID            = "testvault"
-	testVaultIDWithSlashes = "http://example.com/" + testVaultID
-	testDocumentID         = "VJYHHJx4C8J9Fsgz7rZqSp"
-	testDocumentID2        = "AJYHHJx4C8J9Fsgz7rZqSp"
-	testJWE                = `{"protected":"eyJlbmMiOiJjaGFjaGEyMHBvbHkxMzA1X2lldGYiLCJ0eXAiOiJKV00vMS4wIiwiYWxnIjoiQ` +
+	testReferenceID          = "testvault"
+	testVaultIDNoneExistence = "79b2990e-fcf7-11ea-adc1-0242ac120002"
```

Comment:
I think you mean `testVaultIDNonExistent`.

Also, that ID is a valid UUID right? So there's nothing about that string that identifies it as a vault ID that's non-existent... really, what is it is just an example of a UUID/vaultID. I would recommend doing one of the following...

1. Change the name to just `testVaultID`
or
2. Change the vaultID to be something that actually is an impossible vault ID (i.e. not a valid UUID). Then, the "non-existent" name would make more sense.

Reply:
fixed